redwood-tooling-codegen/src/main/kotlin/app/cash/redwood/tooling/codegen/testingGeneration.kt

```diff
@@ -278,24 +279,53 @@ internal fun generateMutableWidget(schema: Schema, widget: Widget): FileSpec {
 
 /*
 public class ButtonValue(
-  public override val modifier: Modifier = Modifier,
-  public val text: String? = null,
-  public val enabled: Boolean = false,
+  override val modifier: Modifier = Modifier,
+  public val text: String?,
+  public val onClick: (() -> Unit)?,
 ) : WidgetValue {
-  public override val childrenLists: List<List<WidgetValue>>
+  override val childrenLists: List<List<WidgetValue>>
     get() = listOf()
 
-  public override fun equals(other: Any?): Boolean = other is SunspotButtonValue &&
+  override fun equals(other: Any?): Boolean = other is SunspotButtonValue &&
     other.text == text &&
     other.enabled == enabled
 
-  public override fun hashCode(): Int = listOf(
+  override fun hashCode(): Int = listOf(
     text,
     enabled,
   ).hashCode()
 
-  public override fun toString(): String =
+  override fun toString(): String =
     """ButtonValue(text=$text, enabled=$enabled)"""
+
+  override fun toDebugString(): String {
+    val debugString = """
+        |ButtonValue(
```

Comment:
Do you think we should keep the "Value` suffix in this rendering? It's useful when referring to it in code, but I think in this rendering we might just want to use the widget name only.